LoopKit/CarbKit/CarbStore.swift

```diff
@@ -185,6 +185,7 @@ public final class CarbStore: HealthKitSampleStore {
      */
     public init(
         healthStore: HKHealthStore,
+        forCurrentAppOnly: Bool,
```

Comment:
nit: Name makes the scoping clear, but not what the scope applies to.  Perhaps `observeHealthKitForCurrentAppOnly`, which uses the existing 'observe' verb pattern to refer to the action of monitoring HealthKit for data.